README.md

```diff
@@ -57,6 +57,11 @@ Then, simply run the following command:
 
 You should now be able to use riscv64-unknown-elf-gcc and its cousins.
 
+NB: Pay attention to the core model of your embedded RISC-V target. Some targets (such as Rocket Chip) 
+use the `medany` core model instead of the default `medlow`, meaning the compiler output produced by 
+the toolchain will be incompatible. To fix this, run the configure script with the `--with-cmodel=medany` option. 
```

Comment:
> meaning the compiler output produced by the toolchain will be incompatible

Assuming that `-mcmodel=medany` is not passed at compile time to override the toolchain's default of `medlow`?